symposion/templates/biblion/atom_feed.xml

```diff
@@ -7,7 +7,7 @@
 
     <title>{{ SITE_NAME }} in Montr&#233;al Announcements</title>
 
-    <link rel="alternate" type="text/html" href="https://us.pycon.org/2015/announcements/" />
+    <link rel="alternate" type="text/html" href="https://us.pycon.org/2016/announcements/" />
```

Comment:
Yes, but there doesn't actually appear to be such a page.  I'll just make it link to home.